src/services/chris_integration.tsx

```diff
@@ -167,18 +166,16 @@ class ChrisIntegration {
         outputFileStem: `${filename}.jpg`, //-slice000
         previous_id: dircopyPluginInstance.data.id
       }
+
       if (imgConverterPlugin === undefined || imgConverterPlugin.data === undefined) {
         return [{
           plugin: this.MED2IMG,
           error: new Error('not registered')
         }];
       }
+
       const imgConverterInstance: PluginInstance = await client.createPluginInstance(imgConverterPlugin.data.id, imgData);
-      console.log("Converter Running")
-      const imgConverterResult = await pollingBackend(imgConverterInstance);
-      if (imgConverterResult.error) {
-        return [imgConverterResult];
-      }
```

Comment:
String interpolation using backtick literals should only be used when necessary in my opinion. In this case, the plugin that gets run could be different depending on whether it's a CT scan or Xray scan.

In the case of `pl-dircopy` and `pl-med2img`, those are less likely to change. By using string interpolation when it's not necessary, it unnecessarily slows down the performance of the portal (which is negligible in smaller apps but as it grows, performance is important). String interpolation less performant because it requires more processing.